src/configuration/interfaces.ts

```diff
@@ -0,0 +1,19 @@
+export interface FilterOption {
```

Comment:
Maybe rename as `interfaces.d.ts`

Reply:
At present, the only TestCafe and third-party libraries  definitions stored in files with `.d.ts` extension. So, I guess we don't change the naming style right now.